contrib/mod_digest.c

```diff
@@ -1871,6 +1871,8 @@ static modret_t *digest_xcmd(cmd_rec *cmd, unsigned long algo) {
   if (pr_fsio_stat(path, &st) < 0) {
     int xerrno = errno;
 
+    pr_log_debug(DEBUG8, MOD_DIGEST_VERSION
+      ": unable to stat %s: %s", path, strerror(xerrno));
```

Comment:
I try to avoid using syscall-specific terminology in log messages; the function `stat(2)` means something to the developer, but not necessarily to the sysadmin/user _reading_ the log messages.  That's why in _e.g._ `mod_site_misc` I use something like:
```
"error checking '%s': %s", path, strerror(xerrno));
```
I'd recommend doing the same for all of these cases you want to add.

Reply:
Make sense. I have changed the log messages in the new commit.  Thank you.